tests/kernel/gen_isr_table/testcase.yaml

```diff
@@ -4,7 +4,7 @@ tests:
     filter: CONFIG_GEN_ISR_TABLES and CONFIG_ARMV6_M_ARMV8_M_BASELINE
     tags: interrupt isr_table
   arch.interrupt.gen_isr_table.arm_mainline:
-    platform_exclude: nucleo_f103rb olimexino_stm32 stm32_min_dev_black
+    platform_exclude: stm32f103_mini nucleo_f103rb olimexino_stm32 stm32_min_dev_black
```

Comment:
Please move this change in a dedicated commit

Reply:
OK